src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -69,12 +69,12 @@
 #include "PointwiseFunctions/AnalyticSolutions/WaveEquation/PlaneWave.hpp"  // IWYU pragma: keep
```

Comment:
I would split this commit into 3:
1. add `energy_density` function and tests
2. add `EnergyDensity` tag and test
3. `EnergyDensityCompute` tags and tests (add into `EnergyDensity.hpp` instead of `Tags.hpp`)
4. update executable